x-pack/plugins/triggers_actions_ui/public/application/lib/rule_api/run_soon.test.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { httpServiceMock } from '@kbn/core/public/mocks';
+import { runSoon } from './run_soon';
+
+const http = httpServiceMock.createStartContract();
+beforeEach(() => jest.resetAllMocks());
+
+describe('disableRule', () => {
```

Comment:
nit: update name of test

Reply:
Resolved in https://github.com/elastic/kibana/pull/139848/commits/40b50cb5653c2e5937115cea9bc2858f4f05cf7a